test/mockRaf.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  allCallbacks: [],
+  prevTime: 0,
+
+  getNow() {
+    return () => {
+      if (this.prevTime === 0) {
+        this.prevTime = Date.now();
```

Comment:
I would avoid Date.now() by all means in tests. Stick to some static value instead. Working with dates we usually use something like `1000` as timestamp. So it is not zero, and not too long to type.